app/steps/success.rb

```diff
@@ -1,8 +1,14 @@
 # frozen_string_literal: true
 
 class Success < Step
-  step_attributes :email
+  step_attributes :email, :faxed_at
 
   validates :email,
     presence: { message: "Make sure to provide an email address" }
+
+  def submitted_date
+    if faxed_at.present?
+      faxed_at.strftime("%m-%d-%Y")
```

Comment:
I think we may want signed_at here? faxed_at will be ~30m from signature

Reply:
Updated at 2240865

This does bring up the issue of not faxing right away. If someone is on the page, they are *done* and we can safely fax their app rather than waiting 30 mins. Thoughts on a separate PR for faxing right away when someone finishes doc upload or indicates that they do not want to do doc upload?